src/openfermion/ops/_polynomial_tensor.py

```diff
@@ -25,10 +25,57 @@ class PolynomialTensorError(Exception):
     pass
 
 
+def general_basis_change(general_tensor, rotation_matrix, key):
+    """Change the basis of an general interaction tensor.
+
+    M' = R^T.M.R where R is the rotation matrix, M is the general tensor
+    and M' is the transformed general tensor.
```

Comment:
Should it be R.M.R^T? Also, perhaps there is a better way to express what this does, as transpose seems to me to be a 2-d concept; I might be wrong.

Reply:
It's a rotation "matrix", of course it's 2-d.

Reply:
Haha ok.